src/main/java/app/model/Review.java

```diff
@@ -13,40 +15,49 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private int id;
-
-    @Column(name = "created_at", updatable = false)
-    @Temporal(TemporalType.TIMESTAMP)
-    @CreationTimestamp
-    private Date createdAt;
-
-    @Column(name = "updated_at")
-    @Temporal(TemporalType.TIMESTAMP)
-    @UpdateTimestamp
-    private Date updatedAt;
+    private Integer id;
 
     @Column(name = "content")
     private String content;
 
-    @ManyToOne(fetch = FetchType.LAZY)
+    @Column(name = "rating")
+    private Integer rating;
+
+    @ManyToOne
     @JoinColumn(name = "user_id")
     private User user;
 
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne
     @JoinColumn(name = "field_id")
     private Field field;
 
-    @OneToMany(fetch = FetchType.LAZY, mappedBy = "review")
-    private List<Like> likes;
+    @ManyToMany(mappedBy = "likedReviews")
+    @Fetch(FetchMode.SUBSELECT)
+    private List<User> likedUsers;
```

Comment:
Sao nó sai sai ri em. Table là likeReview , mà sao tên object là likeUsers nhỉ

Reply:
@vanvtt-0952 đâu cái `mappedBy` là để map tên column mà chị, có phải tên table đâu ạ. Em viết @ManyToMany(mappedBy = "likedReviews") ở đây nghĩa là trường `likedReviews` ở phía bên model của User sẽ map với trường `likedUsers` ở phía model Review ý chị.
Có một [ví dụ khác đây](https://www.baeldung.com/hibernate-many-to-many) ạ.

Reply:
Ok chú